app/src/androidTest/java/com/github/epfl/meili/forum/FirebasePostServiceTest.kt

```diff
@@ -0,0 +1,104 @@
+package com.github.epfl.meili.forum
+
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.internal.runner.junit4.statement.UiThreadStatement
+import com.github.epfl.meili.MainActivity
+import com.github.epfl.meili.forum.Post.Companion.toPost
+import com.google.android.gms.tasks.Task
+import com.google.firebase.firestore.*
+import junit.framework.Assert.assertEquals
```

Comment:
unit.framework.Assert is deprecated, please use org.junit.Assert instead.

Reply:
Ok